src/cudamatrix/cu-allocator.cc

```diff
@@ -185,6 +185,20 @@ void* CuMemoryAllocator::MallocFromSubregion(SubRegion *subregion,
   // region was sufficiently large.  We don't check this; if it segfaults, we'll
   // debug.
 
+  int max_iters=20;
+  int i=0;
+  //search for a block that we don't have to synchronize on
+  auto it = iter;
+  while (it != subregion->free_blocks.end() && i<max_iters) {
```

Comment:
Thanks!  Some small requests:
Space around operators (=20, =0, <max_iters).  Space after //.
No more than 80 chars per line (line break after ||)
Please don't use the name 'it' because it's just a different abbreviation for 'iteration', so it's
unclear how it differs from 'iter'.  I suggest 'chosen_iter'.  Please use int32 as the default int type,
just for consistency with other parts of the code.
I think it would be clearer to make it a for loop.  (Also, pre-decrementing iterators is faster):
```
  for (int32 i = 0; chosen_iter != subregion->free_blocks.end() && i < max_iters;
        ++i,++chosen_iter) {
      // Note: if we reach max_iters and fall off this loop, 'chosen_iter' stays at 'iter', 
      // which will be the smallest block that's big enough.
      ...
```

Reply:
Updated to for loop and changed name. I then ran clang-format with google style on this region.